src/util.cpp

```diff
@@ -19,6 +20,9 @@
 
 void hexDump(const void *memoryLocation, size_t buflen) 
 {
+/*! \def BYTES_PER_LINE
+    \brief A macro that stores the the size of each line of output as a constant 16 bytes
+*/
 #define BYTES_PER_LINE 16
```

Comment:
This doesn't look like it shows up in the generated HTML.

I think doxygen is unhappy with this being inside the `hexDump` function.  Please move the docstring and macro outside of the `hexDump` function.

I also don't think you need the `\def` command.

Please make sure you are checking the generated HTML documentation, not just the source code.

Reply:
Oops, I had checked the HTML documentation the first time, but forgot to recheck when I moved it. I just moved the docstring and macro outside of the hexDump function and committed. It seems to be showing on the HTML documentation now.